p2p/receiver/protocol.go

```diff
@@ -0,0 +1,24 @@
+package receiver
```

Comment:
I created a different package in my pr for some common sender/receiver utils, I will move that there as part of my work.